apps/vercel.mdx

```diff
@@ -260,3 +263,7 @@ module.exports = withAxiom({
 ```
 
 For more configuration options, see the [documentation in the next-axiom GitHub repository](https://github.com/axiomhq/next-axiom).
+
+## Send logs from Vercel preview deployments
```

Comment:
I've moved this section because it seems to be related to the Vercel integration rather than Next.js.

Reply:
looks like my commits got mixed in this PR by mistake.

Reply:
removed them from this PR.